about.html

```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.15.4/css/all.css" integrity="sha384-DyZ88mC6Up2uqS4h/KRgHuoeGwBcD4Ng9SiP4dIRy0EXTlnuz47vAwmeGwVChigm" crossorigin="anonymous">
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <nav class="home-nav">
+        <ul>
+            <li><i class="fab fa-facebook-f"></i></li>
+            <li><i class="fab fa-twitter"></i></li>
+            <li><a href="#">English</a></li>
+            <li><a href="#">My Page</a></li>
+            <li><a href="#">Log Out</a></li>
+        </ul>
+    </nav>
+    <div class="hamb">
+        <i class="fas fa-bars hamburger"></i>
+    </div>
```

Comment:
- [ ] You only implemented the `hamburger `menu but it is not functional. Please make necessary changes so one can redirect between `home `and `about `pages by clicking the links.

Reply:
- [ ] Please fix the `hamburger` menu popup. It is working on the index page, however, it is not functional on the about page. ensure it's working correctly on both pages.